nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -680,6 +680,34 @@ def _impl(inputs, attr, params):
         return _sym.Variable(name=name, shape=params[name].shape)
     return _impl
 
+def _less():
```

Comment:
Could you implement helper function and use it?

for example: 
````python
def _broadcast(name):
    def _impl(inputs, attr, params):
        return AttrCvt(
            op_name=name,
            ignores=['name', 'Tidx'])([inputs[0], inputs[1]], attr)
    return _impl
````

Reply:
Featured in my latest commit